src/App.js

```diff
@@ -1,31 +1,106 @@
+import React, { useState, Component } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import Button from '@material-ui/core/Button';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import TextField from '@material-ui/core/TextField';
+import ROSLIB from 'roslib';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1
-  },
-  title: {
-    flexGrow: 1
+const ros = new ROSLIB.Ros();
+
+class App extends Component {
```

Comment:
Use Hooks instead of class, see the reason in [here](https://reactjs.org/docs/hooks-intro.html).